lib/include/fdoprot.h

```diff
@@ -155,6 +137,7 @@
  *
  * An alternative is to declare the buffer global
  */
+// TODO: Needs review, macro not used.
```

Comment:
should this comment be removed? What is the TODO?

Reply:
This was earlier used to hold content that could be printed on console (nonce, GUID etc.). We've removed those print statements altogether, hence this variable is no longer used. Can be removed.

Reply:
Removed the macro and related comment.

Updated in commit 1c24ccaba0e54c28b18fdfaa30a87b1cad086e07